boards/ti/lp_em_cc2340r5/doc/index.rst

```diff
@@ -0,0 +1,119 @@
+.. _lp_em_cc2340r5:
+
+TI CC2340R5 Launchpad
+#####################
+
+Overview
+********
+
+The Texas Instruments CC2340R5 LaunchPad |trade| (LP_EM_CC2340R5) is a
+development kit for the SimpleLink |trade| multi-Standard CC2340R5 wireless MCU.
+
+See the `TI CC2340R5 LaunchPad Product Page`_ for details.
+
+.. figure:: img/lp_em_cc2340r5.jpg
+   :align: center
+   :alt: TI CC2340R5 Launchpad
+
+   Texas Instruments CC2340R5 LaunchPad |trade|
+
+Hardware
+********
+
+The CC2340R5 LaunchPad |trade| development kit features the CC2340R5 wireless MCU.
+The board is equipped with two LEDs, two push buttons and BoosterPack connectors
+for expansion.
+
+The CC2340R5 wireless MCU has a 48 MHz Arm |reg| Cortex |reg|-M0+ SoC and an
+integrated 2.4 GHz transceiver supporting multiple protocols including Bluetooth
+|reg| Low Energy and IEEE |reg| 802.15.4.
+
+See the `TI CC2340R5 Product Page`_ for additional details.
+
+Supported Features
+==================
+
+The CC2340R5 LaunchPad board configuration supports the following hardware
```

Comment:
put the board name when used with west with \`\` around it and call it board target, not configuration

Reply:
Hello @nordicjm , thanks for the review.
Not sure I understand correctly your request: do you mean that I should refer to the board in this file by using `lp_em_cc2340r5` (between ``) everywhere ? (That's what I understand but I don't find any other example)